src/main.js

```diff
@@ -1,4 +1,14 @@
-const { SigningCosmosClient } = require("@cosmjs/launchpad");
+const {
+    SigningCosmosClient
+} = require("@cosmjs/launchpad");
```

Comment:
It would be good to change`import { SigningCosmosClient } = from '@cosmjs/launchpad'`

Reply:
Fixed, thanks